app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -193,7 +193,11 @@ function AcuantCapture(
                 result = 'success';
               }
 
-              addPageAction('documentCapture.acuantWebSDKResult', { result });
+              addPageAction({
+                key: 'documentCapture.acuantWebSDKResult',
+                label: 'IdV: Acuant web SDK photo analyzed',
```

Comment:
The server already prepends `Frontend: ` so this is going to be `Frontend: IdV: Acuant....` is that OK?

Reply:
> The server already prepends `Frontend: ` so this is going to be `Frontend: IdV: Acuant....` is that OK?

Yeah, at least, I expected it and it seemed reasonable to me.